tools/kms-health-check/src/config.rs

```diff
@@ -0,0 +1,242 @@
+use anyhow::{Context, Result};
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::path::Path;
+
+/// Minimal config structures - only parse what we need for health checks
```

Comment:
Why do we have separate validation here? Can we not re-use existing validation that we do in kms-server?

Reply:
tackled in https://github.com/zama-ai/kms/pull/89/commits/31cb4a99db4671589eaf4358fab986edb032dad3